modules/ROOT/pages/apikit-tutorial.adoc

```diff
@@ -4,16 +4,100 @@ include::_attributes.adoc[]
+
+. Download the example file to your local drive:
++
+`+link:{attachmentsdir}/api.raml[api.raml]+`.
+. Under the File menu, select *New > Mule Project*.
+. In the New Mule Project wizard, enter a Name, myproject for example, Mule Runtime 3.8.0.
++
+. Check Add APIkit Components. In the RAML File field, browse to and select the downloaded `api.raml` file.
++
+image::apikit-tutorial-3e0e4.png[height=633,width=600]
++
+. Click *Finish*.
+
+== Run and Test the Project
+. Right-click the project name in the Project Explorer, then select *Run As > Mule Application*.
++
+The APIkit Console appears below the canvas, displaying the resource-action pairs exposed by the API.
++
+image::apikitconsole-tutorial.png[APIkitconsole-tutorial]
++
+. Click *Introduction* under Documentation to view the API description.
++
+The main Introduction title and sentence map to the contents of `documentation` in the root section of the RAML file.
++
+. Click `/machines` to expand the section. Expand `GET: /machines`, then click the tab labeled *Try It*.
++
+The baseURI is optional in a RAML-backed API, but required by the API Console Try It functionality; otherwise, Try-it is disabled.
++
+image::fv-601.png[]
++
+In the RAML, you can use an arbitrary value, for example `+http://www.example.com+` for the baseURI.
++
+. Click the GET button to send a test call to GET a list of machines.
++
+image::apikitt-get.png[]
++
+. The API returns a list of machines.
++
+The output is the payload set on the message by the `get:/machines:apiConfig` flow.
++
+[source,json,linenums]
+----
+{
+  "count": 3,
+  "machines": [
+    {
+      "id": "ZX4102",
+      "location": "Starbuck's, 442 Geary Street, San Francisco, CA 94102"
+    },
+    {
+      "id": "ZX5322",
+      "location": "Starbuck's, 462 Powell Street, San Francisco, CA 94102"
+    },
+    {
+      "id": "ZX6792",
+      "location": "Cafe La Taza, 470 Post Street, San Francisco, CA 94102"
+    }
+  ]
+}
+----
+
+== Add Logic to the Project
+
+For example purposes, add over-simplified logic to the project to use later in API Manager. Unlike the APIkit-generated flows, the flow you create during this procedure returns a result, a payload message, when you call the API from a browser.
+
+. Drag an HTTP component to the canvas.
++
+Studio creates a flow named apiFlow by default.
++
+. In the apiflow, select the HTTP Connector to open its properties editor.
+. Click image:edit-16x16.gif[Edit-16x16] to edit the Connector Configuration global configuration element.
+. Change the value of the Base Path as follows:
++
+`remote-vending`
++
+. Click OK.
+. In the properties editor, change the Path setting from `/` to `/test-policy`.
```

Comment:
Is the name of the editor actually properties? I don't know if it's named in the UI. If so, match the case of the UI and bold.